.github/workflows/RavenClient.yml

```diff
@@ -17,13 +17,13 @@ jobs:
       RAVENDB_TEST_CLIENT_CERT_PATH: ./certs/nodejs.pem
       RAVENDB_TEST_CA_PATH: /usr/local/share/ca-certificates/ca.crt
       RAVENDB_TEST_HTTPS_SERVER_URL: https://localhost:8989
-      RAVENDB_BUILD_TYPE: stable
+      RAVENDB_BUILD_TYPE: nightly
       RAVEN_License: ${{ secrets.RAVEN_LICENSE }}
     
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x]
-        serverVersion: ["4.2"]
```

Comment:
A 3.x client will NOT work with a 4.0 server since both the client's and the server's Major version do not match

full desc: https://ravendb.net/docs/article-page/4.2/csharp/client-api/faq/backward-compatibility#faq-backward-compatibility

I leave it as it is. We test this client with 5.0 and 5.1 (and 5.2 in near future)

Reply:
ok we add that